Userland/Libraries/LibJS/Tests/eval-basic.js

```diff
@@ -30,3 +30,41 @@ test("returns 1st argument unless 1st argument is a string", () => {
     var stringObject = new String("1 + 2");
     expect(eval(stringObject)).toBe(stringObject);
 });
+var value="outer"
+test("only touches locals if direct access", function() {
```

Comment:
Please use arrow functions for all of these tests to match the rest of LibJS's tests

Reply:
I can't use arrow functions, because arrow functions currently have some kind of incorrect scope behaviour in master, I could add failing tests if there's a way to mark as expected failure?

Reply:
Specifically, even without my change:

   ()=>{eval("var a='wat';")}();

will create a variable a in the global object

Reply:
Then please add a comment explaining exactly that :)

Reply:
Added comment, and filed a bug even :D